src/vm/core.zig

```diff
@@ -37,17 +41,23 @@ pub const CairoVM = struct {
     /// - `CairoVM`: The created VM.
     /// # Errors
     /// - If a memory allocation fails.
-    pub fn init(allocator: *const Allocator) !CairoVM {
+    pub fn init(allocator: *const Allocator, config: Config) !CairoVM {
```

Comment:
add the argument in the doc of the function please

Reply:
sorry missed that.